concept/answer/AnswerGroup.java

```diff
@@ -39,13 +33,6 @@ public AnswerGroup(Concept owner, List<T> answers) {
         this.answers = answers;
     }
 
-    public static AnswerGroup<? extends Answer> of(Transaction tx, AnswerProto.AnswerGroup res) {
```

Comment:
Why is this deleted? We need it to deserialize `ConceptMapGroup` received from the server.

Reply:
(PS I just noticed it is implemented inside `QueryManager` now - we should move it back here for consistency with other `answer` and `Concept` deserializers)